monkey/monkey_island/cc/resources/pba_file_upload.py

```diff
@@ -83,3 +92,12 @@ def delete(self, file_type):
             ConfigService.set_config_value(filename_path, "")
 
         return {}
+
+    @staticmethod
+    def check_file_type(file_type):
+        """
+        Check if the file type is not supported
+        :param file_type: Type indicates which file was received, linux or windows
+        :return: Boolean
+        """
+        return file_type not in {LINUX_PBA_TYPE, WINDOWS_PBA_TYPE}
```

Comment:
Why is `{LINUX_PBA_TYPE, WINDOWS_PBA_TYPE}` a dict?

Reply:
https://docs.python.org/3/tutorial/datastructures.html#sets